notify/notify.yaml

```diff
@@ -375,7 +375,9 @@ components:
             cursors:
               type: object
               properties:
-                $ref: '#/components/schemas/after'
+                after:
```

Comment:
I think you actually want to change the "schema" field to "after" here:

https://github.com/alchemyplatform/docs-openapi-specs/blob/main/notify/notify.yaml#L211

so that anything that references '#/components/schemas/after' also changes

Reply:
yup @yirenlu92 is correct - we want to change the reference schema type here